example-frontend/pages/autoOffsetExactOutETH.tsx

```diff
@@ -21,6 +21,10 @@ export default function AutoOffsetExactOutETH() {
     signer || provider
   );
   const offset = async () => {
+    if (!signer) {
+      alert("Please Connect you Wallet.");
```

Comment:
@GigaHierz can we please get this typo correct to:

```suggestion
      alert("Please connect your wallet first.");
```